src/devtools/client/debugger/src/components/Editor/ToggleWidgetButton.tsx

```diff
@@ -144,7 +146,7 @@ function QuickActions({
       className={classNames(
         "line-action-button absolute z-50 flex translate-x-full transform flex-row space-x-px",
         enableLargeText && "bottom-0.5",
-        hitCounts ? "-right-9" : "-right-1"
+        hitCounts ? (isLineHitCountsCollapsed ? "-right-2" : "-right-5") : "-right-1"
```

Comment:
🥲

Reply:
Yeah, this is something I hope a first-class variants system can make more clear. In this case it would end up looking like:
`<Button x={{ initial: 1, hitCountsEnabled: { initial: 5, collapsed: 2 } }} />`

Reply:
Hmm, I don't think I understand. Why would our design system have awareness of a key like `hitCountsEnabled` ?